roles/openshift_loadbalancer_facts/tasks/main.yml

```diff
@@ -20,6 +20,14 @@
         binds:
         - "*:{{ openshift.loadbalancer.frontend_port }}"
         default_backend: atomic-openshift-api
+      - name: nuage-monitor 
+        mode: tcp
+        options:
+        - tcplog
+        binds:
+        - "*:{{ nuage_mon_rest_server_port }}"
+        default_backend: nuage-monitor
+        when: openshift.common.use_nuage | bool
```

Comment:
Open a PR with some proposed changes for this branch. https://github.com/vishpat/openshift-ansible/pull/1